lib/std/os/linux.zig

```diff
@@ -373,7 +374,7 @@ pub fn faccessat(dirfd: i32, path: [*:0]const u8, mode: u32, flags: u32) usize {
 }
 
 pub fn pipe(fd: *[2]i32) usize {
-    if (comptime builtin.arch.isMIPS()) {
+    if (comptime builtin.arch.isMIPS() || comptime builtin.arch.isSPARC()) {
```

Comment:
I'd put the comptime outside of the parenthesis: `comptime (builtin.arch.isMIPS() || builtin.arch.isSPARC())`

Reply:
I've moved it out, thanks for the suggestion!